### PR TITLE
de-unmaintain-ify

### PR DESCRIPTION
**Pull request by cdaringe**
# problem statement
- it's dead
# solution
- revive it!

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,16 +1,9 @@
...
-
-I’m not using this module any longer and can therefore no longer maintain it.
-If you'd like to adopt this repo, please open a few PRs and I'll happily hand it over. :)
-
 # spawn-pouchdb-server
 
 > Configurable per-app pouchdb-server as a drop-in replacement for CouchDB
 
 [![NPM version](https://badge.fury.io/js/spawn-pouchdb-server.svg)](https://www.npmjs.com/package/spawn-pouchdb-server)
-[![Build Status](https://travis-ci.org/hoodiehq/spawn-pouchdb-server.svg?branch=master)](https://travis-ci.org/hoodiehq/spawn-pouchdb-server)
```
Comment (gr2m) [inline README.md:13]: ahh good catch. Can you change the other other URLS below, too? And also the URLs in package.json?

**Comment by gr2m** [comment]
👍 thanks man! Just left a note on fixing the URLs

**Comment by cdaringe** [comment]
definitely. surprised i didn't see those.  it was late :) 🌚

**Comment by gr2m** [comment]
thanks @cdaringe, I squashed your commits to a `fix: ...` commits so that a new version gets released. That way the README on https://www.npmjs.com/package/spawn-pouchdb-server gets updated.

**Comment by cdaringe** [comment]
oh shucks, you told me already you were doing sem release.  whoops!  need to get in the habit

**Comment by gr2m** [comment]
no worries, the PRs can be as messy as needed, thanks to github we can now squash & rephrase the commits once we are done :)

**Pull request merged**